### DISCUSSION TITLE
Autofill on Vivaldi

### DISCUSSION THREAD

**Discussion by joatca** [zeapo/Android-Password-Store#2308] [Q&A] [answered]
Autofill on Vivaldi looks like it matches only on the Vivaldi app itself and not on the website. Is this fixable or am I doing something wrong?

**Comment by msfjarvis** [comment, marked as answer]
Vivaldi support was only recently added to APS (https://github.com/android-password-store/Android-Password-Store/commit/16b9c1330dbbfb7c0f103a301c344644325e4eda) so it's not going to work on the stable release of the app.

**Reply by joatca** [reply]
Thanks! Good to know that it's in the pipeline. For now I can live with long-press then search, that definitely works.
